app/pages/game.js

```diff
@@ -140,8 +149,8 @@ const PLANET = [
 
 async function init(element) {
   const app = new PIXI.Application({
-    width: 800, // element.clientWidth,
-    height: 600, //element.clientHeight,
+    width: element.clientWidth,
+    height: element.clientHeight,
     backgroundColor: 0x000000, //0x696969,
     resolution: 1,
   });
```

Comment:
how does the height and width of the application relate to the underlying grid?

e.g. if the world is 500x500, and the app is 800x600, what happens? does the edge just get displayed but with no planets in/unmovable? (and how does this adapt to different screen sizes?)